extension/js/common/api/account-servers/external-service.ts

```diff
@@ -150,50 +149,52 @@ export class ExternalService extends Api {
       ),
     });
     const multipartBody = { content, details };
-    return await this.request<FesRes.MessageUpload>('POST', `/api/${this.apiVersion}/message`, {}, multipartBody, { upload: progressCb });
+    return await this.request<FesRes.MessageUpload>(`/api/${this.apiVersion}/message`, { fmt: 'FORM', data: multipartBody }, { upload: progressCb });
```

Comment:
If upload progress needs to be supported for this call, should convert the body to 'JSON' the same way as in `Gmail.msgSend` and make sure FES supports HTTP/2

Reply:
Looks like currently FES supports only HTTP/1.1:

<img width="522" alt="Screenshot 2023-08-23 at 22 39 39" src="https://github.com/FlowCrypt/flowcrypt-browser/assets/1062093/d8cda0d9-ae4d-4220-8bc3-6df266c983ad">

Reply:
Isn't upload progress achievable regardless of http version? We know how much data we intend to send, and we surely know how much data we've sent so far.

Reply:
fetch upload streaming currently has only limited support, only in Chrome > 105 and HTTP/2